sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -636,7 +636,7 @@ result = text_analytics_client.analyze_sentiment(documents, logging_enable=True)
 
 These code samples show common scenario operations with the Azure Text Analytics client library.
 The async versions of the samples (the python sample files appended with `_async`) show asynchronous operations
-with Text Analytics and require Python 3.6 or later.
+with Text Analytics and require Python 3.7 or later.
```

Comment:
You don't need to talk about Python version now, every supported version have async :)